1-exercises/C-highlight-words/script.js

```diff
@@ -1,5 +1,31 @@
 function highlightWords(paragraph, colours) {
```

Comment:
Please check this solution because the paragraph is not rendered. 
![image](https://user-images.githubusercontent.com/54252969/194707643-c3c0977b-6e87-496b-8322-88360bb9b32c.png)